bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -55,10 +55,8 @@ apiServerArguments:
     - "DownwardAPIHugePages=true"
     - "PodSecurity=true"
     - "CSIMigrationAWS=false"
```

Comment:
can you put this file in a separate commit, it's outside the openshift/api revendoring. I would recommend the following:
- commit-1:  bump(openshift/api) - with go.mod
- commit-2: vendor folder, go.sum 
- commit-3: bindata/bootkube/config/bootstrap-config-overrides.yaml

Reply:
Sure, done.